contributors/devel/conformance-guidelines.md

```diff
@@ -0,0 +1,103 @@
+1.9 can be found [here](https://github.com/cncf/k8s-conformance/blob/master/docs/KubeConformance-1.9.md).
+This document will help people understand what features are being tested without having to look through
+the testcase's code directly.
+
+## Adding New Tests
+
+To promote a testcase to the conformance test suite, the following
+steps must be taken:
+- as a prerequisite, the test case is already part of e2e and is not flaky 
+- the testcase must use the `framework.ConformanceIt()` function rather
+  than the `framework.It()` function
+- the testcase must include a comment immediately before the
+  `framework.ConformanceIt()` call that includes all of the required
+  metadata about the test (see the [Test Metadata](#test-metadata) section)
+- use "Promote xxx e2e test to Conformance" as template of your PR title
+- tag your PR with "/area conformance" label
+- send your PR to Sig-Architecture for review by adding "@kubernetes/sig-architecture-pr-reviews" 
+also CC the relevant Sig and Sig-Architecture
+
+
+## Test Metadata
+
+Each conformance test must include the following piece of metadata
+within its associated comment:
+
+- `Release`: indicates the Kubernetes release that the test was added to the
+  conformance test suite. If the test was modified in subsequent releases
+  then those releases should be included as well (comma separated)
+- `Testname`: a human readable short name of the test
+- `Description`: a detailed description of the test. This field must describe
+  the required behaviour of the Kubernetes components being tested using 
+  [RFC2119](https://tools.ietf.org/html/rfc2119) keywords. This field
+  is meant to be a "specification" of the tested Kubernetes features, as
+  such, it must be detailed enough so that readers can fully understand
+  the aspects of Kubernetes that are being tested without having to read
+  the test's code directly. Additionally, this test should provide a clear
+  distinction between the parts of the test that are there for the purpose
+  of validating Kubernetes rather than simply infrastructure logic that
+  is necessary to setup, or clean up, the test.
+
+## Sample Conformance Test
+
+The following snippet of code shows a sample conformance test's metadata:
+
+```
+/*
+  Release : v1.9
+  Testname: Kubelet: log output
+  Description: By default the stdout and stderr from the process being
+  executed in a pod MUST be sent to the pod's logs.
+*/
+framework.ConformanceIt("it should print the output to logs", func() {
+  ...
+})
+```
+
+The corresponding portion of the Conformance Document would then look
+like this:
+
+>
+> ## [Kubelet: log output](https://github.com/kubernetes/kubernetes/tree/release-1.9/test/e2e_node/kubelet_test.go#L47)
+> 
+> Release : v1.9
+> 
+> By default the stdout and stderr from the process being executed in a pod MUST be sent to the pod's logs.
+
+
+Conformance test results, by provider and releases, can be viewed in the 
```

Comment:
This should go under a separate header, it doesn't have anything to do the Sample Conformance Test header above.  Maybe contributing test results with an additional callout to CNCF certification?

Reply:
Fixed.